toolkits/global/packages/global-ad/scss/10-settings/default.scss

```diff
@@ -0,0 +1,12 @@
+$ad--font-family: $font-family-sans;
+$ad--font-size: 1.4rem;
+$ad--leaderboard-bg-color: #ccc;
+$ad--leaderboard-padding: spacing(8);
+$ad--mpu-background-color: #ccc;
+$ad--mpu-margin-top: spacing(24);
+$ad--mpu-padding-top: spacing(8);
+$ad--mpu-padding-bottom: spacing(24);
```

Comment:
Why is padding at the bottom of the mpu so much more than the top?
Was wondering if we could just have `ad--boxed` or something that any ad could use

Reply:
Padding set on `c-ad` with an even 8px all around. Gets reset on skyscraper as that doesn't have a background colour